src/server/database/controllers/messages.js

```diff
@@ -128,7 +128,7 @@ module.exports = {
    * @callback Callbacks when voting attributes have been updated
    */
   vote (req, callback) {
-    return Message.find({where: {id: req.id}})
+    Message.find({where: {id: req.id}})
```

Comment:
Funksjonen har et callback argument som blir kjørt når 'vote' er ferdig. Dette er et alternativ til å returnere et Promise. Er ganske sikker på at den return statementen ikke gjør noe slik den står. Det har uansett ingen påvirkning på hvordan koden blir brukt akkurat nå, siden det ikke blir ventet på at async koden skal kjøre ferdig uansett. (Når man sender melding om å 'vote' i socket håndtering). Jeg trengte det bare for å vite når vote var ferdig i testene mine, for å kunne kjøre testene i riktig rekkefølge.